examples/unfolded/demo_custom_prior_unfolded.py

```diff
@@ -249,7 +249,8 @@ def g(x, *args, **kwargs):
 
 # Get the measurements and the ground truth
 y = physics(test_sample)
-rec = model(y, physics=physics)
+with torch.no_grad():
```

Comment:
why do we need this?

Reply:
because in the forward of unfolded, we can not put a torch.no_grad(), so we need a workaround...